app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,85 @@
+html,
+body {
+  background: #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.